maas-schemas/schemas/maas-backend/subscriptions/subscription.json

```diff
@@ -200,6 +210,18 @@
       },
       "required": ["id"]
     },
+    "benefit": {
```

Comment:
It's not actually benefit provider. It's: _benefit used to purchase subscription_

Reply:
Additionally, at some stage in the past, I and front-end devs agreed to use simply `benefit`. Maybe whim benefits should be renamed to whim benefits :) in order to avoid confusion.